test('hello', () => {


})

test('hellos', () => {

    throw new Error('fail')
})